[2025-10-03T14:47:55.801+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: initial_data_pipeline.load_to_minio manual__2025-10-03T14:47:53.460546+00:00 [queued]>
[2025-10-03T14:47:55.807+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: initial_data_pipeline.load_to_minio manual__2025-10-03T14:47:53.460546+00:00 [queued]>
[2025-10-03T14:47:55.807+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-10-03T14:47:55.818+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_to_minio> on 2025-10-03 14:47:53.460546+00:00
[2025-10-03T14:47:55.824+0000] {standard_task_runner.py:57} INFO - Started process 835 to run task
[2025-10-03T14:47:55.828+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'initial_data_pipeline', 'load_to_minio', 'manual__2025-10-03T14:47:53.460546+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/initial_data_pipeline.py', '--cfg-path', '/tmp/tmpctva2mol']
[2025-10-03T14:47:55.831+0000] {standard_task_runner.py:85} INFO - Job 16: Subtask load_to_minio
[2025-10-03T14:47:55.882+0000] {task_command.py:415} INFO - Running <TaskInstance: initial_data_pipeline.load_to_minio manual__2025-10-03T14:47:53.460546+00:00 [running]> on host 38cb3f89ee79
[2025-10-03T14:47:55.958+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='initial_data_pipeline' AIRFLOW_CTX_TASK_ID='load_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-10-03T14:47:53.460546+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-03T14:47:53.460546+00:00'
[2025-10-03T14:47:56.108+0000] {logging_mixin.py:151} INFO - ✅ Пользователи за 30-09-2025
[2025-10-03T14:47:56.114+0000] {logging_mixin.py:151} INFO - ✅ События за 30-09-2025
[2025-10-03T14:47:56.120+0000] {logging_mixin.py:151} INFO - ✅ Пользователи за 29-09-2025
[2025-10-03T14:47:56.127+0000] {logging_mixin.py:151} INFO - ✅ События за 29-09-2025
[2025-10-03T14:47:56.134+0000] {logging_mixin.py:151} INFO - ✅ Пользователи за 02-10-2025
[2025-10-03T14:47:56.139+0000] {logging_mixin.py:151} INFO - ✅ События за 02-10-2025
[2025-10-03T14:47:56.147+0000] {logging_mixin.py:151} INFO - ✅ Пользователи за 03-10-2025
[2025-10-03T14:47:56.156+0000] {logging_mixin.py:151} INFO - ✅ События за 03-10-2025
[2025-10-03T14:47:56.163+0000] {logging_mixin.py:151} INFO - ✅ Пользователи за 27-09-2025
[2025-10-03T14:47:56.170+0000] {logging_mixin.py:151} INFO - ✅ События за 27-09-2025
[2025-10-03T14:47:56.176+0000] {logging_mixin.py:151} INFO - ✅ Пользователи за 28-09-2025
[2025-10-03T14:47:56.182+0000] {logging_mixin.py:151} INFO - ✅ События за 28-09-2025
[2025-10-03T14:47:56.188+0000] {logging_mixin.py:151} INFO - ✅ Пользователи за 01-10-2025
[2025-10-03T14:47:56.194+0000] {logging_mixin.py:151} INFO - ✅ События за 01-10-2025
[2025-10-03T14:47:56.195+0000] {python.py:194} INFO - Done. Returned value was: ВСЕ данные загружены в MinIO
[2025-10-03T14:47:56.222+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=initial_data_pipeline, task_id=load_to_minio, execution_date=20251003T144753, start_date=20251003T144755, end_date=20251003T144756
[2025-10-03T14:47:56.279+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-10-03T14:47:56.300+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
